src/object/srv_obj_remote.c

```diff
@@ -122,14 +122,13 @@ ds_obj_remote_update(struct dtx_leader_handle *dlh, void *data, int idx,
 	orw_parent = crt_req_get(parent_req);
 	orw = crt_req_get(req);
 	*orw = *orw_parent;
+
 	orw->orw_oid.id_shard = shard_tgt->st_shard_id;
-	uuid_copy(orw->orw_co_hdl, orw_parent->orw_co_hdl);
-	uuid_copy(orw->orw_co_uuid, orw_parent->orw_co_uuid);
```

Comment:
remove the uuid_copy because "*orw = *orw_parent" can copy the uuid?

Reply:
Yes, the uuid_copy is redundant.